### PR TITLE
LPD-36432 Improve range multiselection with keyboard

### PR DESCRIPTION
**Pull request by veroglez**
Improvements have been made to range multiselection. To select multiple items using the keyboard navigation, press `Shift + ArrowUp/ArrowDown`. The items will automatically be selected.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/ControlsContext.js [hunk 1]
```diff
@@ -147,6 +147,14 @@ const reducer = (state, action) => {
 			);
 		}
 		else if (state.multiSelect === MULTI_SELECT_TYPES.range) {
+
+			// Avoid selection in range when directly selecting an item that
+			// is not a layout data item.
+
+			if (!state.layoutData.items[itemId]) {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/ControlsContext.js:154]: When can this happen?

Comment (veroglez) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/ControlsContext.js:154]: I have added in the comment to make it clearer that it's for items such as editables

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**